src/pages/index.js

```diff
@@ -29,54 +31,195 @@ const enableValidation = (config) => {
 
 enableValidation(config);
 
+//VAR
+
+let cardSection;
+let initialUserData;
+
+//API
+
+export const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authorization: "7dcd9a93-149c-4e56-87db-9285c9177a9e",
+});
+
+api
+  .getUserData()
+  .then((data) => {
+    initialUserData = new UserInfo(data);
+  })
+  .catch((err) => console.error(`${err} Profile info may be incorrect`));
+
+api
+  .getInitialCards()
+  .then((data) => {
+    cardSection = new Section(
+      {
+        data: data,
+        renderer: (data) => {
+          const cardElement = createCard(data);
+          cardSection.addItem(cardElement);
+        },
+      },
+      cardListElement
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => console.error(`${err} Cards may be missing`));
+
 const createCard = (cardItem) => {
-  const card = new Card(cardItem, "#card-template", () => {
-    console.log(cardItem);
-    imgPopup.open(cardItem);
-  });
+  const card = new Card(
+    cardItem,
+    "#card-template",
+    () => {
+      imgPopup.open(cardItem);
+    },
+    () => {
+      deleteConfirmPopup.open(card);
+    },
+
+    (likeStatus, id) => {
+      if (!likeStatus) {
+        api
+          .likeCard(id)
+          .then(() => card._handleLikeIcon())
```

Comment:
`_handleLikeIcon`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.

Reply:
Done